tests/test_bash_commands.py

```diff
@@ -3,207 +3,184 @@
 from analysis_driver.util import bash_commands
 from analysis_driver.reader import RunInfo
 
-assets = TestAnalysisDriver.assets_path
-fastq_path = TestAnalysisDriver.fastq_path
-
-
-def test_bcl2fastq():
```

Comment:
These tests had to be moved inside a TestAnalysisDriver object so the config and toolset would be loaded properly.